exporter/prometheusremotewriteexporter/config_test.go

```diff
@@ -76,8 +76,8 @@ func Test_loadConfig(t *testing.T) {
 				WriteBufferSize: 512 * 1024,
 				Timeout:         5 * time.Second,
 				Headers: map[string]string{
-					"prometheus-remote-write-version": "0.1.0",
-					"x-scope-orgid":                   "234"},
+					"Prometheus-Remote-Write-Version": "0.1.0",
+					"X-Scope-OrgID":                   "234"},
```

Comment:
Not sure I understand this change, why is this needed?

Reply:
The provided test config capitalizes its headers and their cases are now preserved: https://github.com/open-telemetry/opentelemetry-collector/blob/6c611f891064fc73c5a3d23019d70e2a69b11591/exporter/prometheusremotewriteexporter/testdata/config.yaml#L19